bundles/org.openhab.binding.dirigera/src/main/resources/json/light-presets/temp-to-color.json

```diff
@@ -0,0 +1,113 @@
+{
```

Comment:
Delete this entirely. The respective table is already in OH Core ColorUtil.

Reply:
removed